File: backend/hrserver/src/main/java/com/beaconfire/hrserver/response/personalinfoResponses/Address.java
package com.beaconfire.hrserver.response.personalinfoResponses;

import com.fasterxml.jackson.annotation.JsonInclude;
import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
@JsonInclude(JsonInclude.Include.NON_NULL)
public class Address {
    //    private ResponseStatus status;
//    private Employee employee;
    private String lineOne;
    private String lineTwo;
    private String city;
    private String state;
    private String zip;
}

File: frontend/hr-client/src/app/pojo/personal-info/address/address.ts
export class Address {
    lineOne : String;
    lineTwo: String;
    city : String;
    state : String;
    zip : String;

    constructor(
        lineOne: String,
        lineTwo : String,
        city : String,
        state : String,
        zip : String
    ){
        this.lineOne = lineOne,
        this.lineTwo = lineTwo,
        this.city = city,
        this.state = state,
        this.zip = zip
    }
}